### ISSUE TITLE
Improve reporting of row count exported

### ISSUE THREAD

**Issue by guy-adams** [transferwise/pipelinewise#251]
https://github.com/transferwise/pipelinewise/blob/6b4c6957763b66b1e760f91241f60d7578df8be4/pipelinewise/fastsync/commons/tap_mysql.py#L192

The way row exports are reported is a little odd. 

For tables where the row count is less than the batch size, it's already as:

```
2019-10-26 21:00:59 INFO: 2019-10-26 21:00:36 - Exported 20000 rows from siemenstestdb.departments...
```

Even if departments only has 10 rows.

For large tables, the log ticks out like:
```
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:12 - Exported 100000 rows from siemenstestdb.employees...
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:21 - Exported 120000 rows from siemenstestdb.employees...
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:30 - Exported 140000 rows from siemenstestdb.employees...
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:39 - Exported 160000 rows from siemenstestdb.employees...
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:49 - Exported 180000 rows from siemenstestdb.employees...
2019-10-26 21:03:39 INFO: 2019-10-26 21:02:58 - Exported 200000 rows from siemenstestdb.employees...
```

But the final row (in this case 200000) is actually the top of the batch range - meaning that the actual number of rows is somewhere between 180000 and 200000. 

I came across this while debugging some target tables that were empty and wanted to see if pipelinewise was actually pulling any rows (and how many).

Looking at the code, I see two options here:

#1 Change: 
`exported_rows += export_batch_rows`
to
`exported_rows += len(rows)`

I've tested this and it works as expected, for small tables:

```
2019-10-26 21:48:24 INFO: 2019-10-26 21:48:20 -Exported  9 rows from siemenstestdb.departments...
```
 
and for large tables

```
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:21 - Exported 20000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:22 - Exported 40000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:22 - Exported 60000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:23 - Exported 80000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:24 - Exported 100000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:25 - Exported 120000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:25 - Exported 140000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:26 - Exported 160000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:27 - Exported 180000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:28 - Exported 200000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:28 - Exported 220000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:29 - Exported 240000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:30 - Exported 260000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:31 - Exported 280000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:32 - Exported 300000 rows from siemenstestdb.dept_emp_latest_date...
2019-10-26 21:52:36 INFO: 2019-10-26 21:52:32 - Exported 300024 rows from siemenstestdb.dept_emp_latest_date...
```

Alternatively, to make it easier to parse the log output, change the wording for "interim" outputs and the "final" output e.g.

For small tables:

```
2019-10-26 21:58:06 INFO:                         ordinal_position
2019-10-26 21:58:06 INFO:
2019-10-26 21:58:06 INFO: 2019-10-26 21:58:02 - Exported total of 9 rows from siemenstestdb.departments...
2019-10-26 21:58:06 INFO: 2019-10-26 21:58:02 - MYSQL - Running query:
```

For large tables:
```
2019-10-26 22:02:50 INFO:
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:33 - Exporting batch from 0 to 20000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:34 - Exporting batch from 20000 to 40000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:34 - Exporting batch from 40000 to 60000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:35 - Exporting batch from 60000 to 80000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:36 - Exporting batch from 80000 to 100000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:37 - Exporting batch from 100000 to 120000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:38 - Exporting batch from 120000 to 140000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:39 - Exporting batch from 140000 to 160000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:39 - Exporting batch from 160000 to 180000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:40 - Exporting batch from 180000 to 200000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:41 - Exporting batch from 200000 to 220000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:42 - Exporting batch from 220000 to 240000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:43 - Exporting batch from 240000 to 260000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:43 - Exporting batch from 260000 to 280000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:44 - Exporting batch from 280000 to 300000 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:44 - Exported total of 300024 rows from siemenstestdb.current_dept_emp...
2019-10-26 22:02:50 INFO: 2019-10-26 22:02:44 - MYSQL - Running query:
```

I can make a pull request for either of these - which do you prefer? My preference is for the later one since if a user only cares about the final row counts it's easier to grep for "Exported total"

**Comment by johnoscott** [comment]
Agree this should be changed and I agree your 2nd option (easier to grep Exported total) is the best option. Thanks for making this change.

**Comment by koszti** [comment]
fixed by #253 . thanks for the this.
